packages/adapters-library/src/adapters/stargate/tests/snapshots/ethereum.withdrawals.farm-v1.json

```diff
@@ -0,0 +1,365 @@
+{
+  "aggregatedValues": ["USD8,061.17"],
+  "latency": "Latency: 2.503 seconds",
+  "snapshot": {
+    "protocolId": "stargate",
+    "name": "Stargate Farm",
+    "description": "Stargate is a fully composable liquidity transport protocol that lives at the heart of Omnichain DeFi",
+    "siteUrl": "https://stargate.finance/",
+    "iconUrl": "https://stargate.finance/favicons/favicon-light.svg",
+    "positionType": "stake",
+    "chainId": 1,
+    "productId": "farm",
+    "success": true,
+    "movements": [
+      {
+        "protocolToken": {
+          "address": "0x38EA452219524Bb87e18dE1C24D3bB59510BD783",
+          "name": "Tether USD-LP",
+          "symbol": "S*USDT",
+          "decimals": 6
+        },
+        "blockNumber": 20661441,
+        "transactionHash": "0xc30e07f1190aa09de896d81a9f44f9755578acf273aee613320ec1ce413a9091",
+        "tokens": [
+          {
+            "address": "0xAf5191B0De278C7286d6C7CC6ab6BB8A73bA2Cd6",
+            "name": "StargateToken",
+            "symbol": "STG",
+            "decimals": 18,
+            "type": "underlying",
+            "blockNumber": 20661441,
+            "balanceRaw": "29168188478059884180158n",
+            "price": 0.27636857144740007,
+            "balance": 29168.188478059885
+          }
```

Comment:
is this a claimed rewards? If so is the type correct? I think it should it be underlying-claimable?

Reply:
I thought so too. But every other adapter implementing the `getRewardWithdrawals` returns them as underlying, so I used this for consistency.